### PR TITLE
Fixes AB#458 Fix dd list when add new address

### PR DESCRIPTION
**Pull request by evgenii-nikulin**

### CODE REVIEW

**Review by yuya-yuliya — APPROVED** [review]

File: src/Feature/Account/client/Integration/Account/sagas.ts [hunk 1]
```diff
@@ -152,14 +152,12 @@ export function* addAddress(action: Action<Commerce.Address>) {
     return yield put(actions.UpdateAddressFailure(error.message, error.stack));
   }
 
-  yield put(actions.UpdateAddressSuccess(data));
+  yield put(actions.AddAddressSuccess(data));
```
Comment (larionovadaria) [inline src/Feature/Account/client/Integration/Account/sagas.ts:155]: Also in this method:
`UpdateAddressFailure --> AddAddressFailure`
`UpdateAddressRequest--> UpdateAddressRequest`

File: src/Feature/Account/client/Integration/Account/reducer.ts [hunk 2]
```diff
@@ -100,7 +107,6 @@ export const saveAddressListReducer = (state: SavedAddressListState = { ...addre
   switch (action.type) {
     case reducerActionTypes.ADDRESS_ADD_FAILURE:
     case reducerActionTypes.ADDRESS_ADD_REQUEST:
-    case reducerActionTypes.ADDRESS_ADD_SUCCESS:
```
Comment (larionovadaria) [inline src/Feature/Account/client/Integration/Account/reducer.ts:103]: Why ?

**Review by ghost — CHANGES_REQUESTED** [review]
Please see my comments

File: src/Feature/Account/client/Integration/Account/reducer.ts (hunk 2 above)
Comment (evgenii-nikulin) [reply src/Feature/Account/client/Integration/Account/reducer.ts:103]: I added another behavior for this action.

**Pull request merged**